resources/migrations/custom_sql/fill_cache_config.my.sql

```diff
@@ -4,8 +4,8 @@ WITH root AS (
          0      AS model_id,
          'ttl'  AS strategy,
          json_object(
-           'multiplier', coalesce((select cast(`value` as unsigned) from setting where `key` = 'query-caching-ttl-ratio'), 10),
-           'min_duration_ms', coalesce((select cast(`value` as unsigned) from setting where `key` = 'query-caching-min-ttl'), 60000)
+           'multiplier', coalesce((select cast(cast(`value` as float) as unsigned) from setting where `key` = 'query-caching-ttl-ratio'), 10),
```

Comment:
Are you sure we want to truncate here, e.g. `2.9` becomes `2`? Can we not rather support a fractional ratio?

Reply:
I don't have strong opinion tbh, this is just follow the existing logic on postgres.